spec/fabricators/comment_fabricator.rb

```diff
@@ -0,0 +1,6 @@
+Fabricator(:comment) do
+  body 'Learn to make burgers, noob'
+  user
+  answer
+  question
```

Comment:
We'll need to keep eyes on this one to mkae sure the polymorphic-ness is working correctly in testing.